themes/wdfn_theme/src/styles/main.scss

```diff
@@ -9,6 +9,17 @@ $fa-font-path: '../fonts' !default;
 @import './academicons';
 
 
+// Workaround for css specificity problem on usa-navbar.
+.usgs-header.usa-header-extended {
+  .usa-navbar {
+    @include at-media($theme-navigation-width) {
+      padding-left: units(4);
+      padding-right: units(4);
```

Comment:
I was going to look this up but can't find it. So does this add 4px of padding?

Reply:
There's already a rule for this, but another identical media query is somehow overriding it with the equivalent of `units(2)`. A unit has a base of 8px, so 4 is 32px. If I understand correctly, if default rem units are used, the 8px would differ depending on the user's default font size.